--- a/vyos-build/packages/linux-kernel/build-linux-firmware.sh	2022-12-31 20:18:03.049039644 +0900
+++ b/vyos-build/packages/linux-kernel/build-linux-firmware.sh	2022-12-31 20:20:28.169952304 +0900
@@ -26,7 +26,7 @@
 
 result=()
 # Retrieve firmware blobs from source files
-FW_FILES=$(find ${LINUX_SRC}/debian/linux-image/lib/modules/${KERNEL_VERSION}${KERNEL_SUFFIX}/kernel/drivers/net -name *.ko | xargs modinfo | grep "^firmware:" | awk '{print $2}')
+FW_FILES=$(find ${LINUX_SRC}/debian/linux-image/lib/modules/${KERNEL_VERSION}-v8${KERNEL_SUFFIX}/kernel/drivers/net -name *.ko | xargs modinfo | grep "^firmware:" | awk '{print $2}')
 
 # Debian package will use the descriptive Git commit as version
 GIT_COMMIT=$(cd ${CWD}/${LINUX_FIRMWARE}; git describe --always)
