--- a/vyos-build/scripts/build-vyos-image
+++ b/vyos-build/scripts/build-vyos-image
@@ -424,7 +424,7 @@
             --bootappend-live "boot=live components hostname=vyos username=live nopersistence noautologin nonetworking union=overlay console=ttyS0,115200 console=tty0 net.ifnames=0 biosdevname=0" \
             --bootappend-live-failsafe "live components memtest noapic noapm nodma nomce nolapic nomodeset nosmp nosplash vga=normal console=ttyS0,115200 console=tty0 net.ifnames=0 biosdevname=0" \
             --linux-flavours {{kernel_flavor}} \
-            --linux-packages linux-image-{{kernel_version}} \
+            --linux-packages linux-image-{{kernel_version}}{{kernel_localversion}} \
             --bootloader {{ bootloaders }} \
             --binary-images iso-hybrid \
             --checksums 'sha256 md5' \
