--- a/vyos-build/data/architectures/arm64.toml
+++ b/vyos-build/data/architectures/arm64.toml
@@ -1,3 +1,10 @@
-# Packages included in ARM64 images by default
-packages = ["grub-efi-arm"]
-bootloaders = "grub-efi"
\ No newline at end of file
+kernel_localversion = "-v8"
+kernel_flavor = "arm64-vyos"
+
+# Packages added to images for ARM64 by default
+packages = [
+  "vyos-linux-firmware",
+  "telegraf",
+  "grub-efi-arm64"
+]
+bootloaders = "grub-efi"
