diff --git a/data/architectures/arm64.toml b/data/architectures/arm64.toml
index ff95d0c..003f417 100644
--- a/data/architectures/arm64.toml
+++ b/data/architectures/arm64.toml
@@ -2,7 +2,8 @@ additional_repositories = [
   "deb [arch=arm64] https://repo.saltproject.io/py3/debian/11/arm64/3005 bullseye main"
 ]
 
-kernel_flavor = "arm64-vyos"
+#kernel_flavor = "arm64-vyos"
+kernel_flavor = "bpi-r3-arm64-vyos"
 
 # Packages included in ARM64 images by default
 packages = ["grub-efi-arm64", "openvpn-dco"]
