diff --git a/packages/linux-kernel/build-jool.py b/packages/linux-kernel/build-jool.py
index 5f26c3b..281a22c 100755
--- a/packages/linux-kernel/build-jool.py
+++ b/packages/linux-kernel/build-jool.py
@@ -4,6 +4,7 @@ from tomllib import loads as toml_loads
 from requests import get
 from pathlib import Path
 from subprocess import run
+import os
 
 def find_arch() -> str:
     tmp=run(['dpkg-architecture', '-q', 'DEB_HOST_ARCH'], capture_output=True)
@@ -36,10 +37,11 @@ KERNEL_SRC: str = Path.cwd().as_posix() + '/linux'
 
 # define variables
 PACKAGE_NAME: str = 'jool'
-PACKAGE_VERSION: str = '4.1.9+bf4c7e3669'
+PACKAGE_VERSION: str = '4.1.10+47334c9124'
 PACKAGE_DIR: str = f'{PACKAGE_NAME}-{PACKAGE_VERSION}'
-SOURCES_ARCHIVE: str = 'jool-4.1.9+bf4c7e3669.tar.gz'
-SOURCES_URL: str = f'https://github.com/NICMx/Jool/archive/bf4c7e3669672367934dc50c8b257b7790f0e27e.tar.gz'
+SOURCES_ARCHIVE: str = 'jool-4.1.10+47334c9124.tar.gz'
+SOURCES_URL: str = f'https://github.com/NICMx/Jool/archive/47334c9124b7a2e3253fb279e6c33acb9c2b09a6.tar.gz'
+#SOURCES_URL: str = f'https://github.com/NICMx/Jool/archive/bf4c7e3669672367934dc50c8b257b7790f0e27e.tar.gz'
 
 # download sources
 sources_archive = Path(SOURCES_ARCHIVE)
@@ -93,6 +95,24 @@ override_dh_auto_install:
 bild_rules = Path(f'{PACKAGE_DIR}/debian/rules')
 bild_rules.write_text(build_rules_text)
 
+#source_format = Path(f'{PACKAGE_DIR}/debian/source/format')
+#source_format.write_text('3.0 (native)')
+print("BOOOOOH!")
+# patch jool
+patches = []
+Path(f'{PACKAGE_DIR}/debian/patches').mkdir(parents=True, exist_ok=True)
+
+for patch in Path('patches/jool').glob('*.patch'):
+    print(f"I: Adding patch {patch}")
+    run(['cp', f'{Path().absolute()}/{patch}', f'{PACKAGE_DIR}/debian/patches/'])
+    patches.append(patch.name)
+    #run(['quilt', 'import', f'{Path().absolute()}/{patch}'], cwd=PACKAGE_DIR)
+    #run(['patch', '-p1', '-i', f'{Path().absolute()}/{patch}'], cwd=PACKAGE_DIR)
+
+series_file = Path(f'{PACKAGE_DIR}/debian/patches/series')
+series_file.write_text('\n'.join(patches))
+#run(['dch', '-n', 'balbalba'], cwd=PACKAGE_DIR)
 # build a package
 debuild_cmd: list[str] = ['debuild']
-run(debuild_cmd, cwd=PACKAGE_DIR)
+#run(debuild_cmd, cwd=PACKAGE_DIR)
+run(['dpkg-buildpackage', '-us', '-uc'], cwd=PACKAGE_DIR)
